src/typeahead/typeahead.ts

```diff
@@ -147,6 +153,10 @@ export class NgbTypeahead implements OnInit,
             this._closePopup();
           } else {
             this._openPopup();
+            if (this.autoSelect !== null && this.autoSelect !== undefined) {
+              this._windowRef.instance.activeIdx = this.autoSelect ? 0 : -1;
+              this._windowRef.instance.autoSelect = this.autoSelect;
```

Comment:
Based on my previous comment I don't think we need `this._windowRef.instance.autoSelect = this.autoSelect`